practice-two/src/style/sections/cta.css

```diff
@@ -0,0 +1,68 @@
+.cta {
+  background-color: var(--color-primary);
+  padding-top: 60px;
+  padding-bottom: 80px;
+}
+
+.cta__wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 20px;
+}
+
+.cta__header {
+  width: 213px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 7px;
+}
+
+.cta__text {
```

Comment:
You can create a common class such as `text-center` to style center. Refer the article https://getbootstrap.com/docs/5.0/utilities/text/